src/helpers/task-metadata.ts

```diff
@@ -2,6 +2,11 @@ import { DateTime } from "luxon";
 import ms from "ms";
 import { ListForOrg, ListIssueForRepo } from "../types/github-types";
 import { ContextPlugin } from "../types/plugin-input";
+import { RestEndpointMethodTypes } from "@octokit/rest";
+
+type IssueLabel = Partial<Omit<RestEndpointMethodTypes["issues"]["listLabelsForRepo"]["response"]["data"][0], "color">> & {
```

Comment:
Why is this needed? You seem to remove color to add it again.

Reply:
The type from the endpoint isn't compatible with the labels being passed through as the type of color is strong | null in it and in this type fetched from endpoint, the type of color is strong | undefined